### ISSUE TITLE
Error when onnx model is fp16

### ISSUE THREAD

**Issue by hotchpotch** [vespa-engine/vespa#30741]
**Describe the bug**

When onnx model is fp16, raise "java.lang.NoClassDefFoundError: ai/onnxruntime/platform/Fp16Conversions"

The reason why the fp16 model is needed is that the 2GB or larger format of onnx does not seem to be viable for vespa at this time. Therefore, highest performance embeddings models (multilingual-e5-large, bge-m3, etc..) are not available. However, if fp16 could be used, it would be possible to run models larger than 2GB, since the file size is almost half of that.

**To Reproduce**

1. set fp16 model in Huggingface Embedder (example [fp16 model is here](https://huggingface.co/hotchpotch/vespa-onnx-intfloat-multilingual-e5-large/blob/main/intfloat-multilingual-e5-large_fp16.onnx), [tokenizer](https://huggingface.co/hotchpotch/vespa-onnx-intfloat-multilingual-e5-large/blob/main/tokenizer.json) )
2. call transformer-model in query
3. An error occurs.

**Exception log**

```
java.lang.Error handling request
exception=
java.lang.NoClassDefFoundError: ai/onnxruntime/platform/Fp16Conversions
   at ai.vespa.modelintegration.evaluator.TensorConverter.toVespaTensor(TensorConverter.java:159)
   at ai.vespa.modelintegration.evaluator.OnnxEvaluator.evaluate(OnnxEvaluator.java:68)
   at ai.vespa.embedding.huggingface.HuggingFaceEmbedder.embed(HuggingFaceEmbedder.java:126)
   at com.yahoo.search.schema.internal.TensorConverter.embed(TensorConverter.java:82)
   at com.yahoo.search.schema.internal.TensorConverter.toTensor(TensorConverter.java:45)
   at com.yahoo.search.schema.internal.TensorConverter.convertTo(TensorConverter.java:35)
   at com.yahoo.search.query.properties.RankProfileInputProperties.set(RankProfileInputProperties.java:46)
   at com.yahoo.processing.request.Properties.set(Properties.java:188)
   at com.yahoo.search.query.properties.PropertyAliases.set(PropertyAliases.java:60)
   at com.yahoo.search.Query.setPropertiesFromRequestMap(Query.java:475)
   at com.yahoo.search.Query.init(Query.java:397)
   at com.yahoo.search.Query.<init>(Query.java:356)
   at com.yahoo.search.Query.<init>(Query.java:340)
   at com.yahoo.search.Query$Builder.build(Query.java:1030)
   at com.yahoo.search.handler.SearchHandler.handleBody(SearchHandler.java:235)
   at com.yahoo.search.handler.SearchHandler.handle(SearchHandler.java:178)
   at com.yahoo.container.jdisc.ThreadedHttpRequestHandler.handle(ThreadedHttpRequestHandler.java:77)
   at com.yahoo.container.jdisc.ThreadedHttpRequestHandler.handleRequest(ThreadedHttpRequestHandler.java:87)
   at com.yahoo.container.jdisc.ThreadedRequestHandler$RequestTask.processRequest(ThreadedRequestHandler.java:191)
   at com.yahoo.container.jdisc.ThreadedRequestHandler$RequestTask.run(ThreadedRequestHandler.java:185)
   at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
   at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
   at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.lang.ClassNotFoundException: ai.onnxruntime.platform.Fp16Conversions not found by model-integration [25]
   at org.apache.felix.framework.BundleWiringImpl.findClassOrResourceByDelegation(BundleWiringImpl.java:1591)
   at org.apache.felix.framework.BundleWiringImpl.access$300(BundleWiringImpl.java:79)
   at org.apache.felix.framework.BundleWiringImpl$BundleClassLoader.loadClass(BundleWiringImpl.java:1976)
   at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
   ... 23 more
```

**Vespa version**

docker image latest (2024/03/27)

```
$ docker inspect --format='{{index .RepoDigests 0}}' vespaengine/vespa
vespaengine/vespa@sha256:09dff03d28ce7587ae2c81a56385f6c930f16aa05b4cae64d694956923aef4ae
```

**Comment by bjormel** [comment]
Thanks for bringing this to our attention, we will look into this next week (Easter break in Norway the rest of this week)

**Comment by jobergum** [comment]
This is fixed in `8.325.46` that was released today. Thank you for your patience!

**Comment by hotchpotch** [comment]
Worked perfectly with `8.325.46` version. Thank you!
